identity-service/src/awsSNS.js

```diff
@@ -58,6 +67,13 @@ function _formatIOSMessage (message, targetARN, playSound = true) {
       }
     }
 
+    if (title !== null) {
```

Comment:
i don't think the !== null is required. if you say 
```js
if (title){
```

it catches null, undefined, falsey values and 0